jax/experimental/jax2tf/tests/primitive_harness.py

```diff
@@ -319,6 +319,40 @@ def parameterized(harness_group: Iterable[Harness],
   ]
 )
 
+lax_top_k = tuple( # random testing
+  Harness(f"_inshape={jtu.format_shape_dtype_string(shape, dtype)}_k={k}",
+          lax.top_k,
+          [RandArg(shape, dtype), StaticArg(k)],
+          shape=shape,
+          dtype=dtype,
+          k=k)
+  for dtype in jtu.dtypes.all #[np.float32, np.int32, np.uint32]
```

Comment:
Can we remove the comment at the end of the line?

Reply:
Addressed in latest commits.